akka-actor-typed-tests/src/test/scala/docs/akka/typed/StyleGuideDocExamples.scala

```diff
@@ -322,7 +322,7 @@ object StyleGuideDocExamples {
 
       sealed trait OperationResult
       case object Confirmed extends OperationResult
-      final case class Rejected(reason: String)
+      final case class Rejected(reason: String) extends OperationResult
```

Comment:
Not necessarily for this PR but we should skip this and use `StatusReply[Done]` instead of defining classes just like it.

Reply:
While I agree that using `StatusReply[Done]` is better suited for this case, for the spirit of the example it seems the current code is more explanatory:
> If several actors share the same message protocol, it’s recommended to define those messages in a separate object for that protocol.

By explicitly creating requests and responses in this separated object, the intent is clearer in my opinion. WDY?

Reply:
Yep, let's leave as is then. Thanks for taking an extra look.

Reply:
Sure thing!